db/knex.js

```diff
@@ -1,3 +1,7 @@
+var debug = require('debug')('app:config')
+
 var env = process.env.NODE_ENV || 'development';
 var config = require('../knexfile.js')[env];
+debug(config);
 module.exports = require('knex')(config);
```

Comment:
@wilmoore 
So after reading the docs about the node debug package I was still pretty confused on how to implement the debugger. I can get the config when I run $ _DEBUG=app:config npm test_ and it shows that it is set to the test-db

_app:config { client: 'pg', connection: 'postgres://localhost/photo_test', pool: { min: 2, max: 10 } } +0ms_

but I do not how to investigate.  I do I get it to freeze?